script/foreman-debug

```diff
@@ -22,7 +22,7 @@ OPTIONS:
   -d DIR  Directory to place the tarball in (default /tmp/foreman-XYZ)
   -g      Skip generic info (CPU, memory, firewall etc.)
   -a      Do not generate a tarball from the resulting directory
-  -m NUM  Maximum lines to keep for each file (default 5000), zero means no limit
+  -m NUM  Maximum size in kB for each file type (default 5000 kB), zero means no limit
```

Comment:
Yes, units are kB, default value 5000. I don't insist on either. Per my comparison of original and new foreman-debug, usually *more* data is collected after the patch applied. Since originally we collected 5k lines (say 500kB) of each file, now around 5MB per each file type - worth comparing tarball of previous and new foreman-debug.

In my opinion, the parameter shall be lower for some log files (i.e. http error log or foreman cron, usually not big) and higher for others (production.log, usually few MBs per day, worth having few days of history). But I dont see an easy and understandable way how to achieve this.